### DISCUSSION TITLE
[FEATURE REQUEST] CFG Modifiers

### DISCUSSION THREAD

**Discussion by darkstorm2150** [hlky/stable-diffusion-webui#207] [Feature suggestions]
It would be great to enable full options on the cfg modifer

I had confirmation for OccultSage dev from stable diffusion that settings range from -40 to 0

What does a negative cfg do?

OccultSage: Negative cfg can be used to guide away from prompt.

OccultSage: But useful for multi prompts.

OccultSage: One prompt guides, other one deflects.

OccultSage: You like fast and furious but hate vin diesel? There you go.

OccultSage: +fast n furious, -vin diesel

=========================

Also 0 CFG creates a random image

I think personally that this would give stable-diffusion-webui a great creative edge over the other github ones.

Thanks!

**Comment by hlky** [comment]
this can be done by just changing cfg scale lower bound to -40 ?

**Reply by darkstorm2150** [reply]
I'm not sure code wise but it should work
